tests/robot-tests/tests/admin_and_public/bau/publish_data.robot

```diff
@@ -241,8 +169,7 @@ Verify newly published release is on Find Statistics page
 
 Navigate to published release page
```

Comment:
As a more general point, we probably shouldn't be having test cases that only call a single keyword. In cases like this, we should probably just delete this test case and put the single keyword in the next test case.